### ISSUE TITLE
`cable-bible` command not longer working

### ISSUE THREAD

**Issue by retokromer** [amiaopensource/cable-bible#50]
@nkrabben reported:

After I `brew install cable-bible`, I try to run it `cable-bible`. I then receive an error.
```
LSCopyApplicationURLsForBundleIdentifier() failed while trying to determine the application with bundle identifier 193-    },.
```
Searching hasn't yielded anything that helps me understand what the issue is.

**Comment by retokromer** [comment]
This line is no longer correct:

https://github.com/amiaopensource/cable-bible/blob/71ca32ef85865f73193e955a2e11e80d317e0789/cable-bible#L7

**Comment by EG-tech** [comment]
fixed by #51, thank you @retokromer!
